CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,543 @@
+---
+CIP: ?
```

Comment:
```suggestion
CIP: 96
```
Please change number now; it'll be updated in repo top-level `README.md` shortly.  Also rename your directory to `CIP-0096` and update the link in your OP which points to the new pathname of the README file in your branch. 🤓